setup.py

```diff
@@ -60,15 +61,15 @@
     url="https://github.com/ApeWorX/eip712",
     include_package_data=True,
     install_requires=[
-        "dataclassy>=0.8.2,<1.0",
-        "eth-utils>=1.3.0,<2",
-        "eth-abi>=2.0.0b7,<3",
-        "eth-typing>=2.2,<3.0",
+        "dataclassy>=0.8.2,<1",
+        "eth-utils>=2.0.0,<3",
```

Comment:
we could try `"eth-utils>=1.0.0,<3",` (etc) to support both; that way it will still work in Ape

Reply:
but I need these eth- prefixed dependencies to be able to handle the newer version _in order to_ get ape upgraded. If we can't upgrade those, we are going to be a tough position eventually as the rest of the internet is moving on from typing-extensions